qprofiler2/src/org/qcmg/qprofiler2/bam/BamSummaryReport2.java

```diff
@@ -314,7 +314,7 @@ private void createPOS(Element parent){
 	 * Parse a SAMRecord Collate various pieces of info from the SAMRecord ready for the summariser to retrieve
 	 * @param record SAMRecord next row in file
 	 */
-	public void parseRecord( final SAMRecord record ) {
+	public void parseRecord( final SAMRecord record ) throws Exception{
```

Comment:
are any exceptions thrown from this method? If not, please remove, otherwise be more specific in terms of the type of exception

Reply:
updated